pkg/scheduler/framework/plugins/dynamicresources/extended/util.go

```diff
@@ -29,11 +29,10 @@ func DeviceClassMapping(draManager fwk.SharedDRAManager) (map[v1.ResourceName]st
 		return nil, err
 	}
 	for _, c := range classes {
-		if c.Spec.ExtendedResourceName == nil {
-			extendedResources[v1.ResourceName(v1beta1.ResourceDeviceClassPrefix+c.Name)] = c.Name
-		} else {
+		if c.Spec.ExtendedResourceName != nil {
 			extendedResources[v1.ResourceName(*c.Spec.ExtendedResourceName)] = c.Name
 		}
+		extendedResources[v1.ResourceName(resourceapi.ResourceDeviceClassPrefix+c.Name)] = c.Name
```

Comment:
A code change in `util.go` with no corresponding test or test change in `util_test.go`? That looks like a gap in unit testing.

Reply:
added checking the implicit extended resource name in the unittest